tests/recovery.py

```diff
@@ -9,13 +9,69 @@
 import ccf.ledger
 import os
 import json
+import random
 from infra.runner import ConcurrentRunner
 from distutils.dir_util import copy_tree
 from infra.consortium import slurp_file
+from e2e_logging import verify_receipt
 
 from loguru import logger as LOG
 
 
+def split_all_ledger_files_in_dir(input_dir, output_dir):
```

Comment:
This was moved to another file in a separate PR - merge conflict?

Reply:
Yep, bad merge it seems.